### PR TITLE
todo-list testing part-1

### PR DESCRIPTION
**Pull request by Abass-rashid**
## Todo-list testing part-1
This is a pair-programming project. in This PR we have created the testing for the todo-list project with jest library for the add and remove functions.

### Highlights
- We have installed jest library
- We have created the addremove.js file to test the add and remove functions

***Cheers and happy reviewing!! :joy_cat:***

### CODE REVIEW

File: src/__tests__/addremove.test.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/**
+ * @jest-environment jsdom
+ */
+// import { newTodo } from '../index.js';
+
```
Comment (tufoinnkuo10) [inline src/__tests__/addremove.test.js]: - [ ] Great job. all your add remove function test are passing. for cleaner code practices, kindly remove the commented code.

File: src/__tests__/addremove.test.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  const newTask = e;
+  const todos = JSON.parse(localStorage.getItem('todos') || '[]');
+
+  const newTodo = {
+    description: newTask,
+    completed: false,
+    id: todos[todos.length - 1] ? todos[todos.length - 1].id + 1 : todos.length + 1,
+  };
+
+  // document.getElementById('new-task').value = '';
```
Comment (tufoinnkuo10) [inline src/__tests__/addremove.test.js]: - [ ] please remove commented code to keep your code clean

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Abass-rashid 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ Test functions for add and remove
✅ Test passing

Required Changes ♻️
Check the comments under the review


tufoin


Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/__tests__/addremove.test.js (hunk 2 above)
Comment (TuyishimireEric) [reply src/__tests__/addremove.test.js]: **Thank you @tufoinnkuo10 for reviewing our codes :+1:**

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi team, 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**